helm_deploy/values-prod.yaml

```diff
@@ -1,178 +1,170 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+generic-service:
+  replicaCount: 4
```

Comment:
Not a huge one but typically people set this in `values.yaml` then need to only override in `values-dev.yaml`